app/controllers/csv_preview_controller.rb

```diff
@@ -1,13 +1,57 @@
+require "csv"
+
 class CsvPreviewController < ApplicationController
   def show
-    legacy_url_path = request.path.sub("/preview", "").sub(/^\//, "")
     @asset = GdsApi.asset_manager.whitehall_asset(legacy_url_path).to_hash
 
+    maximum_columns = 50
```

Comment:
Is there a place where these values could live as constants rather than "magic numbers"? (Maybe there isn't, and this is the only place that will ever need to keep track of them! I have a knee-jerk reaction to numbers in the wild.)

Reply:
Good idea, I'll make them constants.